subsys/nrf_rpc/CMakeLists.txt

```diff
@@ -8,5 +8,9 @@ zephyr_include_directories(include)
 
 zephyr_library()
 
-zephyr_library_sources(nrf_rpc_os.c)
+zephyr_library_sources(
+	nrf_rpc_os.c
+	nrf_rpc_serialize.c
```

Comment:
nrf_rpc_serialize.c and nrf_rpc_cbkproxy.c could be compiled based on additional Kconfig options like NRF_RPC_SERIALIZE_API NRF_RPC_CALLBACK_PROXY